lib/fishjam_web/api_spec/health_report.ex

```diff
@@ -48,12 +36,18 @@ defmodule FishjamWeb.ApiSpec.HealthReport do
     description: "Describes overall Fishjam health",
     type: :object,
     properties: %{
-      status: Status,
-      uptime: %Schema{type: :integer, description: "Uptime of Fishjam (in seconds)"},
-      distribution: Distribution,
-      version: %Schema{type: :string, description: "Version of Fishjam"},
-      gitCommit: %Schema{type: :string, description: "Commit hash of the build"}
+      localStatus: NodeStatus,
+      nodesInCluster: %Schema{type: :integer, description: "Number of nodes in cluster"},
+      distributionEnabled: %Schema{
+        type: :boolean,
+        description: "Cluster distribution enabled/disabled"
+      },
+      nodesStatus: %Schema{
+        type: :array,
+        items: NodeStatus,
+        description: "Status of each node in cluster"
```

Comment:
Shouldn't we mention here that this node is not included?

Reply:
This node is included.